jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/steps/AttachDebugger.kt

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.upload.steps
```

Comment:
Move this out of upload package

Reply:
Yeah all of our steps were in upload but they probably should be in `steps`